google-cloud-clients/google-cloud-datacatalog/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>google-cloud-datacatalog</artifactId>
+  <version>0.87.1-beta-SNAPSHOT</version><!-- {x-version-update:google-cloud-datacatalog:current} -->
```

Comment:
also, inconsistency between alpha and beta here

Reply:
I'll change to alpha.